test/testdrive/consolidation.td

```diff
@@ -109,10 +114,6 @@ $ kafka-ingest format=avro topic=tx schema=${tx-schema}
 # Test that by default updates that occurred at the same time are consolidated,
 # but updates that occurred at distinct times are not.
 
-> CREATE SINK nums_sink FROM nums
```

Comment:
this test was failing on me and it seemed racy (if we create the sink down here then theres still a chance that upper has advanced to 5, which means we won't see the resolution we want)

tbh it is a bit sus that this came out of my timestamping changes, so I could be misreading the issue here